Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -133,17 +133,19 @@ class TopSitesPanel: UIViewController {
         collection?.indexPathsForVisibleItems().forEach(updateRemoveButtonStateForIndexPath)
     }
 
-    private func deleteTileForSuggestedSite(site: SuggestedSite) -> Success {
+    private func deleteTileForSuggestedSite(site: Site) -> Success {
         var deletedSuggestedSites = profile.prefs.arrayForKey("topSites.deletedSuggestedSites") as! [String]
-        deletedSuggestedSites.append(site.url)
+        if !deletedSuggestedSites.contains(dataSource.extractDomainURL(site.url)) {
```

Comment:
o ya this was for the appending suggested sites solution versus replacement so we don't need it anymore